presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -129,6 +129,20 @@ static void addRuntimeMetricIfNotZero(
   }
 }
 
+bool shouldAggregateRuntimeMetric(const std::string& name) {
+  if (name == "dataSourceWallNanos" || name == "dataSourceLazyWallNanos" ||
+      name == "queuedWallNanos") {
+    return true;
+  }
+
+  // 'blocked*WallNanos'
+  if (name.size() > 16 and strncmp(name.c_str(), "blocked", 7) == 0) {
```

Comment:
Can we re here? So it won't break. Do we have any unit test to make sure if won't break? Thanks!